docs/en/observability/tab-widgets/uptime-monitoring/set-up-connect-content.asciidoc

```diff
@@ -0,0 +1,74 @@
+// tag::heartbeat[]
+
+Before we proceed, you'll need to retrieve your Elasticsearch credentials for either an {heartbeat-ref}/configure-cloud-id.html[Elastic Cloud ID] or another {heartbeat-ref}/elasticsearch-output.html[Elasticsearch Cluster].
+
+WARNING: Elastic synthetics runs Chromium without the extra protection of its process https://chromium.googlesource.com/chromium/src/+/master/docs/linux/sandboxing.md[sandbox] for greater compatibility with Linux server distributions.
+Add the `sandbox: true` option to a given browser monitor in {heartbeat} to enable sandboxing.
+This may require using a custom seccomp policy with docker, which brings its own additional risks.
+This is generally safe when run against sites whose content you trust,
+and with a recent version of Elastic synthetics and chromium.
+
+The example below, run from the `examples/todos` directory shows how to run synthetics tests indexing data into Elasticsearch.
+
+Run the script below to start running your synthetics tests. You'll need to insert your actual `cloud.id` and `cloud.auth` values to successfully index data to your cluster.
+
+// NOTE: We do NOT use <1> references in the below example, because they create whitespace after the trailing \
+// when copied into a shell, which creates mysterious errors when copy and pasting!
```

Comment:
Do we really need this? Can't we use `#` comments which wouldn't interfere with it to explain "inline"?

Reply:
I think this note is just here for us to remember not to add `<1>` type references in the future. It's not actually rendered in the docs.